assets/js/admin/specific/webflow/adminDashboardsAddWidget/configWidget/configWidget.js

```diff
@@ -0,0 +1,153 @@
+( function( $ ){
+	// Derived from preside/system/handlers/rules/fieldtypes/TimePeriod.cfc$renderConfiguredField
+	function renderTimePeriodLabel( timePeriod={ type:"alltime" } ) {
+		let i18nArgs = { data:[] };
+		let type     = "alltime"
+		let timeUnit = timePeriod.unit || "d";
+
+		switch( timePeriod.type || "alltime" ) {
+			case "between":
+				type          = timePeriod.type;
+				i18nArgs.data = [ timePeriod.date1 || "", timePeriod.date2 || "" ];
+			break;
+			case "since":
+			case "before":
+			case "until":
+			case "after":
+			case "equal":
+				type          = timePeriod.type;
+				i18nArgs.data = [ timePeriod.date1 || "" ];
+			break;
+			case "recent":
+			case "upcoming":
+			case "pastminus":
+			case "futureplus":
+				type          = timePeriod.type;
+				i18nArgs.data = [
+					  parseInt( timePeriod.measure || 0 )
+					, i18n.translateResource( `cms:time.period.unit.${timeUnit}` )
+				];
+			break;
+			case "pastequal":
+			case "futureequal":
+				type          = timePeriod.type;
+				i18nArgs.data = [
+					  parseInt( timePeriod.measure || 0 )
+					, i18n.translateResource( "cms:time.period.unit.d" )
+				];
+			break;
+			case "future":
+			case "past":
+			case "yesterday":
+			case "today":
+			case "tomorrow":
+			case "lastweek":
+			case "thisweek":
+			case "nextweek":
+			case "lastmonth":
+			case "thismonth":
+			case "nextmonth":
+				type = timePeriod.type;
+			break;
+			default:
+				type = "alltime";
+		}
+
+		return i18n.translateResource( `cms:rulesEngine.time.period.type.${type}.configured`, i18nArgs );
+	}
+
+	var $titleField  = $( "input.admin-dashboards-widget-title" )
+	  , titleBasedOn = $titleField.data( "basedon" )
+	  , $form        = $titleField.closest( "form" );
+
+	if ( titleBasedOn && ( titleBasedOn.length > 0 ) ) {
+		var basedOnFields    = titleBasedOn.split( "|" )
+		  , fieldLabelValues = {}
+		  , regenerateTitle;
+
+		regenerateTitle = function( fields ) {
+
+			for ( var i = 0; i < fields.length; i++ ) {
+				const uberSelect = $( `#${fields[i]}` ).data( "uberSelect" );
+
+				if( uberSelect != undefined ) {
```

Comment:
Should ideally use the strict comparison operator: `if ( uberSelect !== undefined )`.
Reason:

- != performs type coercion, so null != undefined returns false, but other falsy values could give unexpected results.
- !== ensures uberSelect is exactly not undefined (no type coercion), making the code more predictable and less error-prone.